api/install.go

```diff
@@ -4,48 +4,47 @@ import (
 	"encoding/json"
 	"net/http"
 
-	"github.com/gojekfarm/albatross/api/logger"
+	"github.com/gojekfarm/albatross/pkg/helmclient"
+	"github.com/gojekfarm/albatross/pkg/logger"
 )
 
-type InstallRequest struct {
-	Name      string                 `json:"name"`
-	Namespace string                 `json:"namespace"`
-	Chart     string                 `json:"chart"`
-	Values    map[string]interface{} `json:"values"`
-}
-
+// InstallResponse represents the API response to the install request
 type InstallResponse struct {
 	Error  string `json:"error,omitempty"`
 	Status string `json:"status,omitempty"`
+	Data   string `json:"data,omitempty"`
 }
 
 // Install return an http handler that handles the install request
-// TODO: we could use interface as well if everything's in same package
-func Install(svc Service) http.Handler {
+func Install() http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		defer r.Body.Close()
 
-		var req InstallRequest
-		if err := json.NewDecoder(r.Body).Decode(&req); err != nil {
+		operation := helmclient.NewInstallOperation()
```

Comment:
let's keep the contract/struct where the handler exists.  and operation doesn't tell much about what it does/contains.

Depending on protocol http, gRPC, or custom logic the request can differe but helmclient should have minimal struct required for installation